extensions/wikia/PortableInfobox/tests/PortableInfoboxParserTagControllerTest.php

```diff
@@ -129,4 +129,60 @@ public function testMultiWhiteSpacedThemeInfobox() {
 			PortableInfoboxParserTagController::INFOBOX_THEME_PREFIX . $expectedName
 		) );
 	}
+
+	/**
+	 * @dataProvider testGetLayoutDataProvider
+	 */
+	public function testGetLayout( $layout, $expectedOutput, $message ) {
+		$text = '<data><default>test</default></data>';
+
+		$marker = $this->controller->renderInfobox( $text, [ 'layout' => $layout ], $this->parser,
+			$this->parser->getPreprocessor()->newFrame() )[ 0 ];
+		$output = $this->controller->replaceMarkers( $marker );
+
+		$this->assertTrue( $this->checkClassName(
+			$output,
+			$expectedOutput,
+			$message
+		) );
+	}
+
+	public function testGetLayoutNotSet() {
```

Comment:
yes, here we want to call `renderInfobox()` with empty array `[]` and the `testGetLayout()` calls it with `[ 'layout' => $layout ]`